client/stylesheets/_toCat.scss

```diff
@@ -4256,7 +4261,7 @@ On click the images will be displayed at normal size to complete the effect
 
 .footerSocial li {
     display: inline-block;
-    
+    vertical-align: middle !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces